x-pack/plugins/infra/server/lib/alerting/log_threshold/register_log_threshold_rule_type.ts

```diff
@@ -78,6 +78,14 @@ const alertReasonMessageActionVariableDescription = i18n.translate(
   }
 );
 
+const viewInAppUrlActionVariableDescription = i18n.translate(
+  'xpack.infra.logs.alerting.threshold.alertReasonMessageActionVariableDescription',
```

Comment:
it should be `xpack.infra.logs.alerting.threshold.alertViewInAppUrlActionVariableDescription`